### PR TITLE
Feature/change user details

### PR DESCRIPTION
**Pull request by YuliaKr**
Guys,

User details are not available for securityboy as it is not a real user in db. To test functionality please register a new user.

### CODE REVIEW

File: Backend/src/main/java/nl/minor/tourofheroes/backend/controllers/dtos/UserDto.java [hunk 1]
```diff
@@ -3,18 +3,13 @@
 import lombok.Getter;
 import lombok.Setter;
 
-import javax.validation.constraints.NotNull;
-
 @Getter
 @Setter
 public class UserDto {
     //TODO: decide on models, add appropriate validation
```
Comment (Shenaida) [inline Backend/src/main/java/nl/minor/tourofheroes/backend/controllers/dtos/UserDto.java]: make an issue of the TODO for clean code

File: Backend/src/main/java/nl/minor/tourofheroes/backend/services/implementations/UserService.java [hunk 2]
```diff
@@ -111,4 +102,39 @@ public String refresh(String username) {
         //TODO: check when password has been changed
```
Comment (Shenaida) [inline Backend/src/main/java/nl/minor/tourofheroes/backend/services/implementations/UserService.java]: issue aanmaken voor TODO

File: Frontend/src/app/userpages/userdetails/userdetails.component.ts [hunk 3]
```diff
@@ -1,24 +1,63 @@
...
+
+  get f() { return this.userdetails.controls; }
+
+  onSubmit() {
+
+    this.authenticationService.updateUser(this.f.email.value, this.f.lastName.value, this.f.firstName.value)
+      .pipe(first())
+      .subscribe(
+        data => {
+          console.log('----');
```
Comment (Shenaida) [inline Frontend/src/app/userpages/userdetails/userdetails.component.ts]: is deze blijvend nodig?

File: Backend/src/main/java/nl/minor/tourofheroes/backend/controllers/dtos/UserDto.java (hunk 1 above)
Comment (YuliaKr) [reply Backend/src/main/java/nl/minor/tourofheroes/backend/controllers/dtos/UserDto.java]: done

File: Backend/src/main/java/nl/minor/tourofheroes/backend/services/implementations/UserService.java (hunk 2 above)
Comment (YuliaKr) [reply Backend/src/main/java/nl/minor/tourofheroes/backend/services/implementations/UserService.java]: done

File: Frontend/src/app/userpages/userdetails/userdetails.component.ts (hunk 3 above)
Comment (YuliaKr) [reply Frontend/src/app/userpages/userdetails/userdetails.component.ts]: done

**Pull request merged**